contracts/contracts/core/BidderRegistry.sol

```diff
@@ -81,6 +82,8 @@ contract BidderRegistry is
      * @param window The window for which the deposit is being made.
      */
     function depositForWindow(uint256 window) external payable whenNotPaused {
+        require(msg.value != 0, DepositAmountIsZero());
```

Comment:
Do you think Ethereum will allow sending negative values in transfers? It would break backward compatibility for many contracts. 

Also, auditors recommended using `!=` instead of `>`, as it requires less gas.

Reply:
> Do you think Ethereum will allow sending negative values in transfers

Not really, I was thinking that the `window` type (of the `depositForWindow` func) might be changed as part of some refactoring, and so the issue is caught closer to its source. But, as indicated this is a nitpick, so feel free to ignore it.